### PR TITLE
Advance Heart Rate Monitoring System

### PR DESCRIPTION
**Pull request by PasinduDushan**
## Describe your project in 1-3 sentences

I am building an advance heart rate monitoring system with a custom built alert system. Heart rate will be displayed in a nice display with graphs. This project will be really useful for people that is unwell and staying at home.

## Hack Club Slack handle

Pasindu Dushan - U04FJLBBHN2

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

File: PasinduDushan.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 name: "@PasinduDushan"
-project: "Arduino CNC Plotter Machine"
+1st project: "Arduino CNC Plotter Machine"
```
Comment (bellesea) [inline PasinduDushan.md:3]:
```suggestion
project: "Arduino CNC Plotter Machine & Advance Heart Rate Monitoring System With An Alert System"
```

File: PasinduDushan.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
+2nd project: "Advance Heart Rate Monitoring System With An Alert System"
```
Comment (bellesea) [inline PasinduDushan.md:4]:
```suggestion
```

**Review by bellesea — APPROVED** [review]
@PasinduDushan this is super awesome!! thanks for your additional context in the other PR :)

**Comment by PasinduDushan** [comment]
No Problem. I have completed the suggested changes too :)

**Review by MelSmith104 — APPROVED** [review]

**Pull request merged**